daml.yaml

```
@@ -4,7 +4,7 @@
 sdk-version: 2.5.0
 name: daml-finance
 source: src/test/daml
-version: 1.0.4
+version: 1.0.5
```

Shall we bump to
```suggestion
version: 1.1.0
```
? 

We are effectively releasing a new version of the library rather than patching the existing one